src/openhmd.c

```diff
@@ -97,6 +98,18 @@ int OHMD_APIENTRY ohmd_ctx_probe(ohmd_context* ctx)
 	return ctx->list.num_devices;
 }
 
+const char* OHMD_APIENTRY ohmd_gets(ohmd_string_description type)
```

Comment:
This is not conform standars. We are using the return type as int/enum for error logging.
Please rewrite this to use a out pointer like the other get functions.